sites/public/src/components/applications/SubmittedApplicationView.tsx

```diff
@@ -0,0 +1,93 @@
+import { t } from "@bloom-housing/ui-components"
+import { Button, Card, Heading, Icon } from "@bloom-housing/ui-seeds"
+import FormSummaryDetails from "../shared/FormSummaryDetails"
+import { listingSectionQuestions } from "@bloom-housing/shared-helpers"
+import { useMemo } from "react"
+import { DATE_FORMAT } from "../../lib/constants"
+import dayjs from "dayjs"
+import { faChevronLeft } from "@fortawesome/free-solid-svg-icons"
+import { ApplicationSection, Listing } from "@bloom-housing/backend-core/types"
+
+interface SubmittedApplicationViewProps {
+  application: any
```

Comment:
Note that in the cherry-pick, I had to change this to any since the Prisma application type works in this use but not the TypeOrm application type. Due to this, I mirrored the pattern used in FormSummaryDetails but both should be typed once Prisma is in.

Reply:
Can you put that as a comment above it so we remember to change it over to the prisma type when that is pulled over?

Reply:
@ColinBuyck bumping morgan's comment here

Reply:
Done!